Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -1,22 +1,49 @@
+using System.Collections;
 using Unity.Services.Lobbies;
+using UnityEngine;
 
 namespace Lobby
 {
-    public class LobbyManager
+    public class LobbyManager : MonoBehaviour
     {
+        private Unity.Services.Lobbies.Models.Lobby CurrentLobby { get; set; }
+        private string lobbyId;
+        
         public Unity.Services.Lobbies.Models.Lobby CreateLobby()
         {
             var task = Lobbies.Instance.CreateLobbyAsync("MyLobby", 4, new CreateLobbyOptions()
             {
                 IsPrivate = true
             });
-            return task.GetAwaiter().GetResult();
```

Comment:
Do we need to both return a value and also manually set the value of CurrentLobby? One of these two options feels redundant, and the surrounding architecture dictates which.